Makefile

```diff
@@ -91,7 +91,7 @@ api: api-get-apigen
 
 api-models:
 	# Build custom docs
-	php build/docs.php
+	php build/docs.php $(if $(ISSUE-LOGGING-ENABLED),--issue-logging-enabled,)
```

Comment:
Can you please rename ISSUE-LOGGING-ENABLED to ISSUE_LOGGING_ENABLED

more suited for an env variable format.